examples/models/llama/evaluate/eager_eval.py

```diff
@@ -31,7 +31,7 @@ def __init__(
         use_kv_cache: bool = False,
     ):
         device = "cuda" if torch.cuda.is_available() else "cpu"
-        super().__init__(device=device)
+        super().__init__(device=device, pretrained="gpt2")
```

Comment:
Okay, this hack will make the newer version happy by giving it a valid HF model_repo though it won't be used to eval at all. Maybe put a comment for this?